crates/ruff/src/rules/flake8_copyright/settings.rs

```diff
@@ -0,0 +1,83 @@
+// //! Settings for the `flake8-copyright` plugin.
+
+use serde::{Deserialize, Serialize};
+
+use ruff_macros::{CacheKey, CombineOptions, ConfigurationOptions};
+#[derive(
+    Debug, PartialEq, Eq, Serialize, Deserialize, Default, ConfigurationOptions, CombineOptions,
+)]
+#[serde(
+    deny_unknown_fields,
+    rename_all = "kebab-case",
+    rename = "Flake8CopyrightOptions"
```

Comment:
Can we do a brief GitHub Codesearch to see how often (if ever) these options are used? And if they're important to support?

Reply:
From a brief text based search on public repos,

Copyright min file size is used in 70+ places. https://github.com/search?q=copyright-min-file-size&type=code
Copyright regexp is used in 370+ places. https://github.com/search?q=copyright-regexp&type=code
Copyright Author is used in 290+ places https://github.com/search?q=%22copyright-author+%3D+%22&type=code